app/src/main/java/com/example/poste/activities/LoginActivity.java

```diff
@@ -109,7 +109,6 @@ public void onResponse(Call<ResponseBody> call, Response<ResponseBody> response)
                             Log.e("Json Error", err.toString());
                         }
                         // open dashboard activity
-                        User.getUser().updateFoldersAndPosts(LoginActivity.this);
```

Comment:
This is probable caused newer merges happening since you last merged dev in, but if this line is removed dashboard will not have an folders when it launches.

Reply:
I guess I assumed this was occurring in DashboardActivity after a discussion I had with Fayez, but I'll add it back here.